exporter/googlecloudexporter/README.md

```diff
@@ -2,6 +2,113 @@
+
+    ```sh
+    docker pull otel/opentelemetry-collector-contrib
+    ```
+
+    <details>
+    <summary>Alternatives</summary>
+
+    *   Download a [binary or package of the OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases) that is appropriate for your platform, and includes the Google Cloud exporter.
+    *   Create your own main package in Go, that pulls in just the plugins you need.
+    *   Use the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector-builder) to generate the Go main package and `go.mod`.
+
+    </details>
+    
+
+2.  **Create a configuration file `config.yaml`.** The example below shows a minimal recommended configuration that receives OTLP and sends data to GCP, in addition to verbose logging to help understand what is going on. It uses application default credentials (which we will set up in the next step).
+
+    Note that this configuration includes the recommended `memory_limiter` and `batch` plugins, which avoid high latency for reporting telemetry, and ensure that the collector itself will stay stable (not run out of memory) by dropping telemetry if needed.
+
+    ```yaml
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    exporters:
+      googlecloud:
+      logging:
+        loglevel: debug
+    processors:
+      memory_limiter:
+      batch:
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [memory_limiter, batch]
+          exporters: [googlecloud, logging]
+        metrics:
+          receivers: [otlp]
+          processors: [memory_limiter, batch]
+          exporters: [googlecloud, logging]
+    ```
+
+3.  **Set up credentials.**
+
+    1.  Enable billing in your GCP project.
+
+    2.  Enable the Cloud Metrics and Cloud Trace APIs.
+
+    3.  Ensure that your user GCP user has (at minimum) `roles/monitoring.metricWriter` and `roles/cloudtrace.agent`. You can learn about [metric-related](https://cloud.google.com/monitoring/access-control) and [trace-related](https://cloud.google.com/trace/docs/iam) IAM in the GCP documentation.
+
+    4.  Obtain credentials.
+
+        ```sh
+        gcloud auth application-default login
+        ```
+
+    <details>
+      <summary>Alternatives</summary>
+
+      * You can run the collector as a service account, as long as it has the necessary roles. This is useful in production, because credentials for a user are short-lived.
+
+      * You can also run the collector on a GCE VM or as a GKE workload, which will use the service account associated with GCE/GKE.
+    </details>
+
+4.  **Run the collector.** The following command mounts the configuration file and the credentials as Docker volumes. It runs the collector in the foreground, so please execute it in a separate terminal.
+
+    ```sh
+    docker run \
+      --volume $(pwd)/key.json:/etc/otel/key.json \
```

Comment:
I think this would be in their config directory now (this might be a more accurate way to get it `gcloud --format='value(config.paths.global_config`)

```suggestion
      --volume ~/.config/gcloud/application_default_credentials.json:/etc/otel/key.json \
```